elm-sdks/morphir-unit-test/src/Morphir/UnitTest/Expect.elm

```diff
@@ -0,0 +1,120 @@
+module Morphir.UnitTest.Expect exposing(..)
+import Morphir.SDK.Dict exposing(Dict, toList, get)
+import Morphir.SDK.List exposing(List)
+import Morphir.SDK.Set exposing(Set)
+
+type Expectation = Expectation ExpectationResult
+type ExpectationResult = Pass | Fail String
+
+getResult : Expectation -> ExpectationResult
+getResult Expectation result = result
+
+getResultBool : Expectation -> Bool
+getResultBool e = case e of
+    Expectation Pass -> True
+    Expectation (Fail _) -> False
+
+equal : a -> a -> Expectation
```

Comment:
Should "a" be "comparable" instead? In proper Elm I think it should be, but I don't know whether Morphir maintains this concept.

Reply:
Good point, tho in morphir it's "eq" not "comparable" apparently

Reply:
Actually on further investigation I think `==` in elm doesn't need to be comparable; elm code will compile with that called on a general type parameter